Skype/Trusted-Application-API/SDK/ClientModel/Resources/AudioVideoFlow.cs

```diff
@@ -91,6 +91,28 @@ public FlowState State
             return await tcs.Task.ConfigureAwait(false);
         }
 
+        public async Task StopPromptsAsync(LoggingContext loggingContext)
```

Comment:
Following things are also needed to make it usable:
1. Add this method to IAudioVideoFlow interface
2. Add a corresponding capability in AudioVideoCapabilities
3. Update Supports method in AudioVideoFlow class to handle the new capability
4. Add unit tests